components/inputs/input-checkbox.js

```diff
@@ -210,7 +210,7 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 					aria-label="${ifDefined(this.ariaLabel)}"
 					@change="${this.#handleChange}"
 					class="d2l-input-checkbox"
-					@click="${this._handleClick}"
+					@click="${this.#handleClick}"
```

Comment:
Sorry @margaree this will conflict with your other PR that removes this.

Reply:
No worries!